packages/database/src/migrations/20200912130823-AddMoreLaosSchoolsSchoolIndicatorsEiEMapOverlays-modifies-data.js

```diff
@@ -0,0 +1,230 @@
+'use strict';
```

Comment:
This migration is for adding some new overlays to School Indicators EiE group:

> Additional reading materials received since June 2020 (Responses: Yes, No, No data) (Code: SchCVD004b)
> Access to water supply all year round (Responses: Yes, No, No data) (Code: SchCVD010l)
> Lao satellite receiver and dish set (Yes, No, No data) (Code: SchCVD012b)
> School implementing MoES safe school protocols/guidelines (Responses: Yes, fully. Yes, partially. No. No data) (Code: SchCVD027)
> Access to clean drinking water (Responses: Tap water, Bottle water provided by school, Filtered or boiled water, Clean water from home, No, No data) (Code: To combine SchCVD028 and SchCVD029)